particle_filter_node:
  ros__parameters:
    num_particles: 600
    
    motion_noise_x: 0.05
    motion_noise_y: 0.05
    motion_noise_theta: 0.1
    
    alpha1: 0.1   
    alpha2: 0.1   
    alpha3: 0.1   
    alpha4: 0.1   
    
    sensor_noise: 0.2
    max_range: 10.0
    min_range: 0.1
    
    likelihood_field_sigma: 0.5
    z_hit: 0.95
    z_rand: 0.05
    
    resample_threshold: 0.5
    
    update_rate: 10.0
    laser_subsample: 10
